ci-operator/step-registry/azure/bastion/pre/azure-bastion-pre-ref.yaml

```diff
@@ -0,0 +1,25 @@
+ref:
+  as: azure-bastion-pre
+  from_image:
+    namespace: ocp
+    name: "4.10"
+    tag: upi-installer
+  grace_period: 10m
+  commands: azure-bastion-pre-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
```

Comment:
The bastion ref is only used for provision/deprovision bastion host, should not touched with any install-config.yaml, so I think these parameters are not needed here. I also gave the same comments in gcp cases.